crawler/file_processing.py

```diff
@@ -426,11 +422,10 @@ def backup_file(self) -> None:
         shutil.copyfile(self.full_path_to_file(), destination)
 
     def create_import_record_for_file(self) -> None:
-        logger.info(f"{self.docs_inserted} documents inserted")
```

Comment:
Do we no longer want to log in this method?

Reply:
At first I didn't think it was too helpful but with more context I think it will be.

Reply:
I'm going to move this logging to somewhere more appropriate, in the function that gets called.